plugins/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -287,7 +292,7 @@ def main():
         zone_name=dict(type='str', required=True),
         record_name=dict(type='str', aliases=['name'], required=True),
         record_type=dict(type='str', default='A', choices=record_types),
-        record_value=dict(type='str', required=True),
+        record_values=dict(type='list', elements='str', required=True, aliases=['record_value']),
```

Comment:
```suggestion
        record_value=dict(type='str', aliases=['record_value']),
        record_values=dict(type='list', elements='str', aliases=['record_value']),
```
You also need to add `mutually_exclusive` and `required_one_of` to make sure that both can be specified, but at least one of them must be. Then you need code which makes `record_values` a one-element list with the value of `record_value` if the latter is specified to the module and the former is not.